.gitlab-ci.yml

```diff
@@ -15,8 +14,7 @@
       tailpid=$!
       sinfo -l | grep $SLURM_Q
 
-      #test build script 
-      jobid=$(sbatch --export=ALL -A EXASGD --gres=gpu:1 --ntasks=1 -p $SLURM_Q -o output -e output -t 60:00 $WORKDIR/buildsystem/build.sh $SCRIPT_ARGS)
+      #test build script
```

Comment:
Did you delete the test script? Make sure there is actually a real test being run here before merging, but having a smoke test of a pipeline might be good for debugging secrets...

Reply:
Good Idea. I will make a little smoke test for debugging secrets in the event something goes wrong.

Reply:
This seems to be the last unresolved thread before we can experiment with getting secrets to work

Reply:
I am currently testing on the gitlab side first.